service-front/assets/js/moj/moj.modules/moj.cookie-consent.js

```diff
@@ -11,6 +11,21 @@
                 window.GOVUK.cookie('cookie_policy') || window.GOVUK.setDefaultConsentCookie()
             }
 
+            if (this.isInCookiesPage()) {
+                var noJsMessage = document.querySelector('.no-js');
```

Comment:
I think this probably won't have any effect. If JS is available, there won't be a no-js class, but a js-enabled one. So I think finding and removing this is redundant.

Reply:
Tested this locally, it is needed if we want the no js message appearing. The function in main only removes it from the body element & not from anything else.